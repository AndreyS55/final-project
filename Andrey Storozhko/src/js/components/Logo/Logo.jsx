import React from 'react';
import { Link } from 'react-router-dom'
import logo from '../../../img/Logo.png';
import styles from './Logo.scss';

const Logo = () => (
  <Link to='/'>
    <img src={logo} className={styles.header__logo} alt="logo"/>
  </Link>
);

export default Logo;