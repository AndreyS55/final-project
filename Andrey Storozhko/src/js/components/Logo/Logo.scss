@import '../../../styles/environment/colors';

.headerLogo {
  display: flex;
  justify-content: center;
  align-items: center;
}

.logoIcon {
  height: 50px;
  margin-left: 15px;
}

.logoTitle {
  margin-left: 15px;
  color: $wht;
}
