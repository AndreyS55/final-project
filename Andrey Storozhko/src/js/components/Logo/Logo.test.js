import React from 'react';
import Logo from './Logo';

it('should render a Logo', () => {
  const component = shallow(<Logo />);
  expect(component).toMatchSnapshot();
});
