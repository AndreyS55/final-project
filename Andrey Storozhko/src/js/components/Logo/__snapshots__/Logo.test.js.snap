// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a Logo 1`] = `
<Link
  className="headerLogo"
  replace={false}
  to="/"
>
  <img
    alt="logo"
    className="logoIcon"
    src="test-file-stub"
  />
  <h1
    className="logoTitle"
  >
    Pokedex
  </h1>
</Link>
`;
